### PR TITLE
Screen/login

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added login screen

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        class="nav bg-dark-2 l-2 r-2 t-2  p-2 pl-3 pr-3 h-auto fixed row items-center z-50 br-lg b-solid b-1 h-min br-primary back-blur-5 bx-sh-primary-3">
+        <nav class="row items-center justify-between w-full ">
+            <ul class="left row items-center">
+                <h1 class="text-lg text-light">ZenCart</h1>
+            </ul>
+              <ul class="right row items-center flex-gap-1 text-light">
+                <li><a href="index.html">Home</a></li>
+                <li><button class="btn btn-light">
+                        <a href="login.html">
+                            Login</a></button></li>
```
Comment (deekshithmd) [inline login.html]: check alignment of tags

**Review by deekshithmd — APPROVED** [review]

File: login.html (hunk 1 above)
Comment (sanjitsarkar) [reply login.html]: I have used prettier extension still it's not fixing it.

**Review by ritikaawasthy — APPROVED** [review]
good to merge

**Review by manish-GH — APPROVED** [review]
The code and auth pages look great, just one suggestion

For the login page on the last line add 'SignUp'
As in don't have an account? SignUp

Similarly for signup page add 'Login'

**Pull request merged**